CODE_OF_CONDUCT.md

```diff
@@ -7,7 +7,7 @@ We as members, contributors, and leaders pledge to make participation in our
 community a harassment-free experience for everyone, regardless of age, body
 size, visible or invisible disability, ethnicity, sex characteristics, gender
 identity and expression, level of experience, education, socio-economic status,
-nationality, personal appearance, race, religion, or sexual identity
+nationality, personal appearance, race, caste, color, religion, or sexual identity
```

Comment:
No need to change anything here, these two are already covered under the existing code of conduct

Reply:
These two words were added in the 2.1 version:
https://www.contributor-covenant.org/version/2/1/code_of_conduct/

Reply:
Ah ok, nevermind then! 👍